### PR TITLE
RadixSort fixed bug, adding test file for it 

### PR DESCRIPTION
**Pull request by Teuton-Roy**
- Fix one bug; sorting doesn't work for empty array.
- Add test cases for Radix_sort.

- [x] This pull request is all my own work -- I have not plagiarized it.
- [x] I know that pull requests will not be merged if they fail the automated tests.
- [x] This PR only changes one algorithm file. To ease review, please open separate PRs for separate algorithms.
- [x] All new Java files are placed inside an existing directory.
- [x] All filenames are in all uppercase characters with no spaces or dashes.
- [x] All functions and variable names follow Java naming conventions.
- [x] All new algorithms have a URL in their comments that points to Wikipedia or other similar explanations.
- [x] If this pull request resolves one or more open issues then the commit message contains `Fixes: #{$ISSUE_NO}`.

### CODE REVIEW

**Review by debasishbsws — APPROVED** [review]
Everything looks good.

File: src/main/java/com/thealgorithms/sorts/RadixSort.java [hunk 1]
```diff
@@ -2,61 +2,79 @@
...
-    private static int getMax(int[] arr, int n) {
-        int mx = arr[0];
-        for (int i = 1; i < n; i++) {
+    /*
+     * This method returns the maximum value in the array
+     * @param arr The array where to find the maximum value
+     */
+    private int getMax(int[] arr) {
+        int mx = Integer.MIN_VALUE;
+        for (int i = 0; i < arr.length; i++) {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/sorts/RadixSort.java:23]: Please throw an exception if array is empty

File: src/main/java/com/thealgorithms/sorts/RadixSort.java [hunk 2]
```diff
@@ -2,61 +2,79 @@
...
-    static void print(int[] arr, int n) {
+    public static void main(String[] args) {
+        int[] arr = { };
+        int n = arr.length;
+        RadixSort rs = new RadixSort();
+        rs.radixSort(arr);
         for (int i = 0; i < n; i++) {
             System.out.print(arr[i] + " ");
         }
     }
```
Comment (siriak) [inline src/main/java/com/thealgorithms/sorts/RadixSort.java:79]: Please remove `main()`

**Review by siriak — CHANGES_REQUESTED** [review]

**Review by siriak — CHANGES_REQUESTED** [review]
The code looks good, please fix PR checks

File: src/main/java/com/thealgorithms/sorts/RadixSort.java [hunk 3]
```diff
@@ -2,61 +2,76 @@
...
-
-    public static void main(String[] args) {
-        int[] arr = { 170, 45, 75, 90, 802, 24, 2, 66 };
-        int n = arr.length;
-        radixsort(arr, n);
-        print(arr, n);
-    }
-}
-// Written by James Mc Dermott(theycallmemac)
+
```
Comment (debasishbsws) [inline src/main/java/com/thealgorithms/sorts/RadixSort.java:77]:
```suggestion
    }
```

You Have Missed a Curly Braces here

**Review by debasishbsws — CHANGES_REQUESTED** [review]

**Review by debasishbsws — DISMISSED** [review]

File: src/main/java/com/thealgorithms/sorts/RadixSort.java [hunk 4]
```diff
@@ -2,61 +2,76 @@
...
-        int m = getMax(arr, n);
-
-        for (int exp = 1; m / exp > 0; exp *= 10) {
-            countSort(arr, n, exp);
-        }
-    }
+    /**
+     * @param arr
+     */
+    public void radixSort(int[] arr) {
```
Comment (siriak) [inline src/main/java/com/thealgorithms/sorts/RadixSort.java:66]: Please handle empty array. It's already sorted

**Review by siriak — CHANGES_REQUESTED** [review]

**Pull request closed without merging**